.github/workflows/publish.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     - name: Set version
       run: |
         sed -i -e "s/0.0.0/${{ steps.previoustag.outputs.tag }}/" package.json
+        sed -i -e "s/0.0.0/${{ steps.previoustag.outputs.tag }}/" package-lock.json
```

Comment:
`package-lock.json` also carries the version for `gettyimages-api` so it seems correct to also update that.